packages/conf-sdl2-image/conf-sdl2-image.1/opam

```diff
@@ -7,7 +7,11 @@ depexts: [
   ["sdl2_image-dev"] {os-distribution = "alpine"}
   ["libsdl2-image-dev"] {os-family = "debian"}
   ["libsdl2-image-dev"] {os-family = "ubuntu"}
+  ["SDL2_image-devel"] {os-distribution = "fedora"}
+  ["epel-release" "SDL2_image-devel"] {os-distribution = "centos"}
+  ["SDL2_image-devel"] {os-distribution = "ol"}
   ["libsdl2_image-devel"] {os-distribution = "mageia"}
+  ["libsdl2_image-devel"] {os-family = "suse"}
```

Comment:
I think for opensuse it should be "libSDL2_image-devel"

Reply:
https://opensuse.pkgs.org/15.2/games-x86_64/libSDL2_image-devel-2.0.5-lp152.35.9.x86_64.rpm.html